stempy/reader.cpp

```diff
@@ -23,6 +26,11 @@ using std::vector;
 
 namespace stempy {
 
+const int SECTOR_WIDTH = 144;
```

Comment:
It would be good to add a brief comment explaining why these are hard-coded for our future selves.